modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepConfigTable.scala

```diff
@@ -54,8 +54,8 @@ object StepConfigTable {
 
   private def columns(p: Props): List[Table.ColumnArg] =
     List(
-      Column(Column.props(p.size.width.toInt / 2, "name", label = "Name")),
-      Column(Column.props(p.size.width.toInt / 2, "value", label = "Value"))
+      Column(Column.props(p.size.width.toInt / 2, "name", label = "Name", flexShrink = 0, flexGrow = 0, className = SeqexecStyles.paddedStepRow.htmlClass)),
+      Column(Column.props(p.size.width.toInt / 2, "value", label = "Value", flexShrink = 0, flexGrow = 0, className = SeqexecStyles.paddedStepRow.htmlClass))
```

Comment:
So is the default value for `flexShrink` and `flexGrow` nonzero, then?

Reply:
yes, `flexShrink` is 1 which breaks alignment with the styling we are using

Reply:
I mean `flexShrink` is `1` by default on the upstream library